### PR TITLE
Switch date of runs to time_start and show execution time

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Improves https://github.com/web-platform-tests/wpt.fyi/issues/204 and adds run execution-time data for a misinterpretation of https://github.com/web-platform-tests/wpt.fyi/issues/282

Changes the hover content to show how long a run took (and what time it started). This disambiguates the date shown.

### CODE REVIEW

**Comment by foolip** [comment]
Cool! The staging instance still shows different dates, but it's probably less likely to happen this way.

**Comment by foolip** [comment]
In other words, I don't think this should close https://github.com/web-platform-tests/wpt.fyi/issues/204, but perhaps reduce the priority.

File: webapp/components/test-run.html [hunk 1]
```diff
@@ -92,7 +93,29 @@
...
+          hour: 'numeric',
+          minute: 'numeric',
+        });
+        return `${date} at ${time}`;
+      }
+
+      timeTaken(testRun) {
+        const hour = 1000*60*60;
+        const elapsed = new Date(testRun.time_end) - new Date(testRun.time_start);
+        return `${Math.round(elapsed / hour * 10) / 10}h`;
```
Comment (foolip) [inline webapp/components/test-run.html]: Maybe use .toFixed(1) here?

**Comment by foolip** [comment]
Oh, actually this doesn't totally fix https://github.com/web-platform-tests/wpt.fyi/issues/282 either, that was about exposing test-level duration, which we have in the reports now.

Still, this PR is great :)

**Comment by Hexcles** [comment]
Unfortunately, the execution time isn't always reliable.

Some runners do not provide `time_start` and `time_end`, in which case the results receiver will fill the current timestamp into these two fields, essentially making `time_start` the same as `created_at` and causing the execution time to be almost 0.

One solution is to change the results receiver to stop faking `time_end` and having the frontend to hide execution time if `time_end` is null.

**Comment by lukebjerring** [comment]
I'll leave it to you to tweak this PR then :)

**Pull request merged**